gotocompany/guardian/v1beta1/guardian.proto

```diff
@@ -524,6 +525,14 @@ message ListUserApprovalsRequest {
   repeated string statuses = 2;
   repeated string order_by = 3;
   string account_id = 4;
+  uint32 size = 5 [(validate.rules).uint32 = {
+    gte: 0,
+    ignore_empty: true
+  }];
+  uint32 offset = 6 [(validate.rules).uint32 = {
+    gte: 0,
+    ignore_empty: true
+  }];
```

Comment:
should we do validation from the proto or in the guardian service itself?

Reply:
I think validation still can be added here, but we also should validate in the guardian service layer

Reply:
I have used this since this felt like a kind of proto field documentation. Don't have a preference either way. @rahmatrhd @utsav14nov What do you suggest ?